docs/levenscyclus/index.md

```diff
@@ -1,32 +1,37 @@
 icon: material/reload
 --- 
+# Levenscyclus algoritmes en AI
 
-!!! info "Disclaimer"
+Verantwoordelijke algoritmes en AI voldoen tijdens elke fase van de levenscyclus aan de regels. Van het nemen van een besluit tot het stoppen met algoritmes of AI.
 
-    Het Algoritmekader is nog volop in ontwikkeling. Op deze plek willen we vooral aan de slag gaan op een open en transparante wijze. Het is dus niet definitief. Dat betekent dat er dingen opstaan die niet af zijn en soms zelfs fout. Mocht er iets niet kloppen, laat het ons weten via [GitHub](https://github.com/MinBZK/Algoritmekader).
+## Fases van de levenscyclus
+1.	[Probleem analyseren](https://minbzk.github.io/Algoritmekader/levenscyclus/probleemanalyse/)
+2.	[Ontwerpen](https://minbzk.github.io/Algoritmekader/levenscyclus/ontwerp/)
+3.	[Data onderzoeken en voorbereiden](https://minbzk.github.io/Algoritmekader/levenscyclus/dataverkenning-en-datapreparatie/)
+4.	[Ontwikkelen](https://minbzk.github.io/Algoritmekader/levenscyclus/ontwikkelen/)
+5.	[Beoordelen](https://minbzk.github.io/Algoritmekader/levenscyclus/verificatie-en-validatie/)
+6.	[Implementeren](https://minbzk.github.io/Algoritmekader/levenscyclus/implementatie/)
+7.	[Controle en onderhoud](https://minbzk.github.io/Algoritmekader/levenscyclus/monitoring-en-beheer/)
+8.	[Stoppen](https://minbzk.github.io/Algoritmekader/levenscyclus/uitfaseren/)
 
-Algoritmes en AI kunnen door overheidsorganisaties worden gebruikt, bijvoorbeeld bij het leveren van diensten of het nemen van besluiten. 
-Algoritmes en AI-systemen doorlopen een zogeheten levenscyclus. 
-Een algoritme wordt ontwikkeld, in productie genomen, en na enige tijd van gebruik kan worden besloten het gebruik ervan te beëindigen. 
-Omdat de levenscyclus van een algoritme en AI aanzet tot nadenken over de inzet van algoritmes van begin tot eind is het bruikbaar als leidraad om relevante informatie te structureren en te communiceren. 
-Wat is het doel, gaat het gebruik ervan wel naar verwachting, wanneer moet wat worden gedaan en wat als het gebruik ten einde loopt? 
-In praktijk kan het zo zijn dat de levenscyclus anders verloopt, bijvoorbeeld omdat na validatie en verificatie eerst terug naar de tekentafel moet worden gegaan (ontwerpfase) omdat het nog niet aan de wensen of vereisten voldoet.  
+De fases in deze levenscyclus passen bij de manier van werken van overheden. Andere fases zijn ook mogelijk.
 
-  ![Levenscyclus](../afbeeldingen/levenscyclus/levenscyclus.jpg)
+## Bronnen
+- CRISP-DM (cross-industry standard process for data mining)
+- ASUM-DM (analytics solutions unified method)
+- SEMMA (Sample, Explore, Modify, Model, and Assess)
+- Microsoft TDSP (Team Data Science Process)
+- MDLM (mobile device lifecycle management)
+- 5 sleutel [?]
+- Gemeente Amsterdam [?]
+- NIST (National Institute of Standards and Technology)
+- ISO 22989
+- BZK – vanaf gebruiker [?]
+- Onbekende bron [?]
+- Onbekende bron [?]
```

Comment:
wat is dit?

Reply:
Ipv deze lijst zou ik uitleggen dat we verschillende levenscycli hebben gebruikt (noem een paar van de bronnen), en op basis daarvan tot 1 levenscyclus zijn gekomen die zo goed mogelijk aansluit bij de praktijk.

Reply:
Zie eerdere suggestie.